Tests/SwiftAlgorithmsTests/ChunkedTests.swift

```diff
@@ -76,4 +76,105 @@ final class ChunkedTests: XCTestCase {
     XCTAssertLazySequence(fruits.lazy.chunked(by: { $0.first == $1.first }))
     XCTAssertLazySequence(fruits.lazy.chunked(on: { $0.first }))
   }
+  
+  
+  //===----------------------------------------------------------------------===//
+  // Tests for `chunks(ofCount:)`
+  //===----------------------------------------------------------------------===//
+  func testChunksOfSize() {
+    XCTAssertEqualSequences([Int]().chunks(ofCount: 1), [])
+    XCTAssertEqualSequences([Int]().chunks(ofCount: 5), [])
+
+    let collection = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10]
+    XCTAssertEqualSequences(collection.chunks(ofCount: 1),
+                            [[1], [2], [3], [4], [5], [6], [7], [8], [9], [10]])
+    XCTAssertEqualSequences(collection.chunks(ofCount: 3),
+                            [[1, 2, 3], [4, 5, 6], [7, 8, 9], [10]])
+    XCTAssertEqualSequences(collection.chunks(ofCount: 5),
+                            [[1, 2, 3, 4, 5], [6, 7, 8, 9, 10]])
+    XCTAssertEqualSequences(collection.chunks(ofCount: 11),
+                            [[1, 2, 3, 4, 5, 6, 7, 8, 9, 10]])
+  }
+  
+  func testChunksOfSizeBidirectional() {
+    let collection = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10]
+
+    XCTAssertEqualSequences(collection.chunks(ofCount: 1).reversed(),
+                            [[10], [9], [8], [7], [6], [5], [4], [3], [2], [1]])
+    XCTAssertEqualSequences(collection.chunks(ofCount: 3).reversed(),
+                            [[10], [7, 8, 9], [4, 5, 6], [1, 2, 3]])
+    XCTAssertEqualSequences(collection.chunks(ofCount: 5).reversed(),
+                            [[6, 7, 8, 9, 10], [1, 2, 3, 4, 5]])
+    XCTAssertEqualSequences(collection.chunks(ofCount: 11).reversed(),
+                            [[1, 2, 3, 4, 5, 6, 7, 8, 9, 10]])
+  }
+  
+  func testChunksOfSizeCount() {
+    XCTAssertEqual([Int]().chunks(ofCount: 1).count, 0)
+    XCTAssertEqual([Int]().chunks(ofCount: 5).count, 0)
+
+    let collection = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10]
+    XCTAssertEqual(collection.chunks(ofCount: 1).count, 10)
+    XCTAssertEqual(collection.chunks(ofCount: 3).count, 4)
+    XCTAssertEqual(collection.chunks(ofCount: 5).count, 2)
+    XCTAssertEqual(collection.chunks(ofCount: 11).count, 1)
+  }
+  
+  func testChunksOfSizeDistance() {
```

Comment:
You should be able to replace these last two tests with a call to `validateIndexTraversals`.

Reply:
Done!